src/events/utilities/grade-calculator/grade-calculator.ts

```diff
@@ -1,41 +1,56 @@
-import { LowPassFilter } from './low-pass-filter';
+import { isNumber } from '../helpers';
+
+const KalmanFilter = require('kalmanjs');
+
+export const CLAMP = 40
```

Comment:
Why not 45 :) ?

Reply:
Good catch. Well I looked at the strava files for the GAP calculator. 
I took the first few samples where grade should be >45% and strava reports 28%. 
No matter how much smoothing I would put, it would not drop this to be compliant. 
Looking at the strava streams I could not find a value of >40% 

However, I might be wrong. 

What do you think?

Reply:
I'm ok with this. I was just suprised^^. Testing the PR is long. I have difficulties to be sure results match expectations.

Reply:
Take your time. No rush